### PR TITLE
added fix to make sure ids are not none

### PR DESCRIPTION
**Pull request by jgao1995**
#240 
added a checker to make sure the id is not None

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -360,7 +361,8 @@ def secret_key(self):
 
     @models.permalink
     def get_absolute_url(self):
-        return ('app.views.event_show', [str(self.id)])
+        if (self.id != None):
```
Comment (dlakata) [inline app/models.py]: not too familiar with the code, but I'm pretty sure you can do ```if self.id:``` instead.

Comment (adelq) [inline app/models.py]: Yes, in Python the idiomatic way is to use `if self.id:`. Please fix

**Comment by adelq** [comment]
LGTM after fix

**Comment by adelq** [comment]
LGTM

**Pull request merged**